tools/sense_studio/sense_studio.py

```diff
@@ -44,6 +48,9 @@
 
 socketio.init_app(app)
 
+# Training script expects videos in MP4 format
+VIDEO_EXT = '.mp4'
```

Comment:
It looks like `VIDEO_EXT` could be used in more places... See other comments below: